desktop/mac/release-notes/index.md

```diff
@@ -33,6 +33,43 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### Security
+
+- Update Docker Engine to v20.10.14 to address [CVE-2022-24769](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-24769)
+- Update containerd to v1.5.11 to address [CVE-2022-24769](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-24769)
+
+### New
+
+- IT Administrators can now install Docker Desktop remotely using the command line. For more information, see [Install Docker Desktop using the command line](../install.md#install-from-the-command-line).
+- Add [Docker SBOM plugin v0.6.0](https://github.com/docker/sbom-cli-plugin/releases/tag/v0.6.0)
```

Comment:
Do we have any documentation about what this is or how people should use it?

Reply:
We have a PR for SBOM updates - https://github.com/docker/docker.github.io/pull/14516